pkg/security/podsecuritypolicy/provider_test.go

```diff
@@ -482,8 +488,8 @@ func TestValidatePodSecurityContextSuccess(t *testing.T) {
 		Level: "level",
 	}
 
-	seccompNilWithNoAnnotations := defaultPod()
-	seccompNilWithNoAnnotationsPSP := defaultPSP()
+	seccompNilWithDefaultAnnotation := defaultPodWithSeccomp()
```

Comment:
Why don't we use "no annotation" for default anymore? That would also solve the docker-vs-rkt-vs-... problem.